### PR TITLE
Add more information to Readme

### LINKED ISSUE #1: Please add instructions how to use it in readme

### PR DESCRIPTION
**Pull request by rubo77**
closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
 # translate
-A [maubot](https://github.com/maubot/maubot) to translate words.
+A [maubot](https://github.com/maubot/maubot) to translate words using google-tranlslator (DeepL is planned too)
```
Comment (tulir) [inline README.md]:
```suggestion
A [maubot](https://github.com/maubot/maubot) to translate words using Google Translate (DeepL is planned too)
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+    !translate en ru Hello world.
+    
+which results in
+
+    Translate Bot:
+    > rubo77
+    > !translate en ru Hello world.
+    Привет, мир.
+    
+supported_languages:
```
Comment (tulir) [inline README.md]: https://cloud.google.com/translate/docs/languages

**Review by tulir — CHANGES_REQUESTED** [review]
Thanks! Needs a few changes to fix that first line and add a link to the full list of supported languages

File: README.md (hunk 2 above)
Comment (rubo77) [reply README.md]: what do you want to say with that link? are those all supported?

**Comment by rubo77** [comment]
feel free to cherry pick my commit and use it as a base for a more thoroughly change of the readme.

File: README.md (hunk 2 above)
Comment (tulir) [reply README.md]: Yes, it mostly just uses Google Translate directly

**Comment by rubo77** [comment]
fixed, I also added a hint to the alias `!tr`

**Comment by rubo77** [comment]
bump

**Pull request merged**